frontend/src/ui/components/PreviewDropdown/PreviewDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { Collapse, Nav } from 'react-bootstrap';
+import { WrappedMessage } from 'utils/intl';
+import messages from './displayMessages';
+import './styles.scss';
+
+interface PreviewDropdownProps {
+  children?: React.ReactNode;
+  initialExpanded?: boolean;
```

Comment:
Code below doesn't use these props. Do we need them to be defined?

Reply:
We don't need them; they're from an early implementation. Had I kept the git history it would be obvious.
Thanks for your remarks.